internal/appsec/config.go

```diff
@@ -53,15 +61,18 @@ func newConfig() (*config, error) {
 		log.Info("appsec: starting with the security rules from file %s", filepath)
 	} else {
 		log.Info("appsec: starting with the default recommended security rules")
+		cfg.rules = []byte(staticRecommendedRule)
 	}
 
-	cfg.wafTimeout = 4 * time.Millisecond
-	if wafTimeout := os.Getenv("DD_APPSEC_WAF_TIMEOUT"); wafTimeout != "" {
+	cfg.wafTimeout = defaultWAFTimeout
+	if wafTimeout := os.Getenv(wafTimeoutEnvVar); wafTimeout != "" {
 		timeout, err := time.ParseDuration(wafTimeout)
-		if err != nil {
+		if err == nil {
```

Comment:
the reason why configuring the waf timeout was broken: this condition was wrong ^^'